hydra-access-controls/lib/hydra/config.rb

```diff
@@ -22,13 +24,22 @@ def [] key
           permissions
         when :user_model
           user_model
+        when :id_to_resource_uri
+          id_to_resource_uri
         else
           raise "Unknown key #{key}"
       end
     end
 
     attr_reader :permissions
-    attr_accessor :user_model
+    attr_accessor :user_model, :id_to_resource_uri
+
+    # This is purely used for translating an ID to user-facing URIs not used for
+    # persistence. Useful for storing RDF in Fedora but displaying their
+    # subjects in content negotiation as local to the application.
+    def id_to_resource_uri
+      @id_to_resource_uri ||= lambda { |id| ActiveFedora::Base.id_to_uri(id) }
```

Comment:
Change the `lambda { ... }` to `AF::Base.translate_id_to_uri` Which is a lambda already